bundles/org.openhab.binding.bluetooth.hdpowerview/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>4.3.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.bluetooth.hdpowerview</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: HD Powerview Bluetooth Adapter</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.bluetooth</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
```

Comment:
The Capabilities Database class contains information about different types of shades from Hunter Douglas plus their respective capabilities. This file will be updated from time to time as HD introduces new models. So rather than having the file with two clone copies in different locations for different bindings, and having the risk of the two versions being out of synch, IMHO it is better that the Capabilities Database is maintained in just one binding, and shared across both bindings.

Reply:
Sounds totally reasonable. But just want to check with other @openhab/add-ons-maintainers if there are other options and if this implementation is ok

Reply:
@jlaur wdyt?

Reply:
I don't think I've been around for long enough. 😄 I saw this: https://github.com/openhab/openhab-addons/pull/17326#issuecomment-2308717612

But I don't know if this rule is about runtime dependencies only (i.e. one binding requires another binding to work), or if it also restricts from having this kind of dependency. @J-N-K, @kaikreuzer - perhaps you can advise?

Reply:
Just for clarification: my original code in this binding is not trying to depend on another binding; it is importing one single Java class file from another binding..

Reply:
As this seems to be a different type of dependency and nobody is really against i'll tend to merge it as is.

Only thing that bothers me is that this dependency knowledge might get lost.  Could you somehow add a comment to the source file? And double check, the build will fails as this file is no longer available, right?

Reply:
> Could you somehow add a comment to the source file?

I will add the comment in this PR to the respective source file in the other binding.

> And double check, the build will fails as this file is no longer available, right?

Yes.